src/policy/immix/immixspace.rs

```diff
@@ -529,6 +513,21 @@ impl<VM: VMBinding> ImmixSpace<VM> {
         did_defrag
     }
 
+    pub fn clear_side_log_bits(&self) {
```

Comment:
Now this method is called in both `GenImmix::prepare` and `StickyImmix::prepare`, but not within `ImmixSpace::prepare` or `ImmixSpace::release`.  This is not a problem because the plan may need more control about which space to prepare and which metadata to clear.  But we may need to take a note because we want a systematic approach to clearing metadata.  https://github.com/mmtk/mmtk-core/issues/1256